pkg/render/fluentd.go

```diff
@@ -1026,12 +1027,12 @@ func (c *fluentdComponent) fluentdClusterRole() *rbacv1.ClusterRole {
 			ResourceNames: []string{c.fluentdName()},
 		})
 	}
-	if c.cfg.Installation.KubernetesProvider == operatorv1.ProviderOpenShift {
+	if c.cfg.Installation.KubernetesProvider.IsOpenShift() {
 		role.Rules = append(role.Rules, rbacv1.PolicyRule{
 			APIGroups:     []string{"security.openshift.io"},
 			Resources:     []string{"securitycontextconstraints"},
 			Verbs:         []string{"use"},
-			ResourceNames: []string{PSSPrivileged},
+			ResourceNames: []string{securitycontextconstraints.Privileged},
```

Comment:
It looks consistent with the other code, but is it really necessary? Could it have been the hostaccess one? I'm ok postponing that work to another ticket though.